include/signal.h

```diff
@@ -234,6 +234,14 @@
 #  endif
 #endif
 
+/* SIGEXIT is to trigger atext/on_exit logic */
+
+#ifndef CONFIG_SIG_SIGEXIT
+#  define SIGEXIT       18
```

Comment:
should we use SIGKILL for this?

Reply:
Did you mean move default SIGKILL to user space, and redirect all task exit request to it ?

Reply:
Yes. BTW, to follow the strict user/kernel separation, it's better to move all default signal handler to userspace side.